cassiopeia/data/utilities.py

```diff
@@ -165,4 +174,131 @@ def _compute_dissimilarity_map(cm, C, missing_state_indicator, nb_weights):
 
         return dm
 
-    return _compute_dissimilarity_map(cm, C, missing_state_indicator, nb_weights)
+    return _compute_dissimilarity_map(
+        cm, C, missing_state_indicator, nb_weights
+    )
+
+
+def sample_bootstrap_character_matrices(
+    character_matrix: pd.DataFrame,
+    prior_probabilities: Optional[Dict[int, Dict[int, float]]] = None,
+    B: int = 10,
```

Comment:
I'd prefer a lowercase and more expressive name than `B`, e.g. `n_bootstraps` or `num_bootstraps` etc.